Header header
int32  left_motor_temperature
int32  right_motor_temperature
int32  left_driver_temperature
int32  right_driver_temperature
bool   tilt_driver_overheat
bool   height_driver_overheat
